### PR TITLE
Use Perspective for Athena results

### PR DESCRIPTION
**Pull request by fiskus**
* Replaced custom M.Table layout with Perspective renderer
* Change the data format of Athena results to one consumed by Perspective

### CODE REVIEW

File: catalog/app/components/Preview/renderers/Perspective/Perspective.tsx [hunk 1]
```diff
@@ -199,10 +198,9 @@ const useStyles = M.makeStyles((t) => ({
 export interface PerspectiveProps
   extends React.HTMLAttributes<HTMLDivElement>,
     PerspectiveOptions {
-  data: string | ArrayBuffer
-  meta: ParquetMetadata
-  handle: S3HandleBase
-  onLoadMore: () => void
+  data: perspective.PerspectiveInput
+  meta?: ParquetMetadata
+  onLoadMore?: () => void
```
Comment (fiskus) [inline catalog/app/components/Preview/renderers/Perspective/Perspective.tsx:203]: I already treat them as optional

File: catalog/app/containers/Bucket/Queries/Athena/Athena.tsx [hunk 2]
```diff
@@ -682,10 +682,12 @@ export default function Athena({
                 {queryResultsData.case({
                   Init: () => null,
                   Ok: (queryResults: requests.athena.QueryResultsResponse) => {
-                    if (queryResults.list.length) {
+                    if (queryResults.rows.length) {
                       return (
                         <Results
-                          results={queryResults.list}
+                          className={classes.results}
```
Comment (fiskus) [inline catalog/app/containers/Bucket/Queries/Athena/Athena.tsx:688]: Fixed margin btw

**Review by nl0 — APPROVED** [review]
looks good, i hope it works too =)

**Comment by fiskus** [comment]
Tested

**Pull request merged**